Dockerfile

```diff
@@ -0,0 +1,9 @@
+# escape=`
+
+FROM microsoft/windowsservercore
+
+COPY .\src\Lighthouse\bin\Release C:\lighthouse\
+COPY .\src\Lighthouse.WhatsMyIP\bin\Release C:\lighthouse.whatsmyip\
+
+RUN ["C:\\lighthouse\\Lighthouse.exe","install","--localsystem","--autostart"]
```

Comment:
It's relative for anyone building the project on their host machine, these are telling where the executable goes inside of the docker image once it's built.  So basically the paths are absolute *inside the image, but relative outside.